R/utils.R

```diff
@@ -150,3 +150,38 @@ get_teal_bs_theme <- function() {
     bs_theme
   }
 }
+
+#' Format expression to string
+#' Convert any expression to a single character vector
+#' @param code (`language`, `expression`, `character`)
+#' @return `character(1)`
+#' @keywords internal
+format_expression <- function(code) {
```

Comment:
Why redefine function in this package rather than use `teal.code:::format_expression`?